<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AssetCheckUI</class>
 <widget class="QMainWindow" name="AssetCheckUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,2,1,2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Level : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="level_comboBox"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Lod : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="lod_comboBox"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string>This scene</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="listWidget"/>
    </item>
    <item>
     <widget class="QLabel" name="info_label">
      <property name="text">
       <string>UV data</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>374</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
